mewarpx/scripts_ancillary/tinyprofiler_plotting_example.py

```diff
@@ -42,13 +42,12 @@
 import json
 import os
 import math
+from pathlib import Path
 
-import matplotlib.pyplot as plt
+from matplotlib import pyplot as plt, cm
```

Comment:
I'm surprised that syntax even works... but let's use simpler syntax :)
```suggestion
from matplotlib import pyplot as plt
from matplotlib.pyplot import cm
```